src/brokers/commerzbank.js

```diff
@@ -0,0 +1,327 @@
+import format from 'date-fns/format';
+import parse from 'date-fns/parse';
+import Big from 'big.js';
+
+import { parseGermanNum, validateActivity } from '@/helper';
+
+// These functions are for single transaction documents
+const findAmountBuy = textArr => {
+  const amountIndex = textArr.findIndex(t => t.includes('IBAN')) - 1;
+  return parseGermanNum(textArr[amountIndex]);
+};
+
+const findAmountDividend = textArr => {
+  const amountIndex =
+    textArr.findIndex(t => t.includes('Steuerbemessungsgrundlage')) - 1;
+  return parseGermanNum(textArr[amountIndex]);
+};
+
+const findAmountForeignDividend = textArr => {
+  const amountIndex = textArr.findIndex(t => t.includes('Devisenkurs:')) + 4;
+  return parseGermanNum(textArr[amountIndex]);
+};
+
+const findSharesBuy = textArr => {
+  return Big(
+    parseGermanNum(textArr[textArr.findIndex(t => t.includes('St.')) + 1])
+  );
+};
+
+const findSharesDividend = textArr => {
+  return Big(
+    parseGermanNum(textArr[textArr.findIndex(t => t.includes('Stk.')) + 1])
+  );
+};
+
+const findSharesForeignDividend = textArr => {
+  return Big(
+    parseGermanNum(textArr[textArr.findIndex(t => t.includes('STK')) + 1])
+  );
+};
+
+const findDateBuy = textArr => {
+  const date_string =
+    textArr[textArr.findIndex(t => t.includes('Verwahrungs-Art:')) - 3];
```

Comment:
That will result for the first buy test case the date `06.01.2020` instead of the order date:
>    "GESCHÄFTSABRECHNUNG",
    "VOM",
    "03.01.2020",

```suggestion
    textArr[textArr.findIndex(t => t.includes('GESCHÄFTSABRECHNUNG')) + 2];
```

Reply:
Fixed